### DISCUSSION TITLE
Update offer - mandatory fields

### DISCUSSION THREAD

**Discussion by MKI-Miro** [allegro/allegro-api#6846] [Suggestions - rejected]
Hi

What are the mandatory fields when trying to update offer price or stock amount? (PATCH https://api.{environment}/sale/product-offers/{offerId})

Thank you

**Comment by AureliuszBrussy** [comment]
The Discussions tab is used for suggestions. I have converted your submission to [an Issue](https://github.com/allegro/allegro-api/issues/6847). I answered your question there.
